samwise.js

```diff
@@ -7,19 +7,17 @@
 /* global clearTimeout, setTimeout, document */
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?